Raspberry/lantronixScales/html/index.html

```diff
@@ -0,0 +1,73 @@
+<html>
+<header>
+  <title>Lantronix Scale Reader</title>
+  <link rel="stylesheet" href="bootstrap/bootstrap.min.css">
+</header>
+<body>
+  <nav class="navbar navbar-dark bg-dark">
+    <a class="navbar-brand" href="#">CamScripter</a>
+  </nav>
+
+  <div class="jumbotron">
+    <div class="container">
+      <h1 class="display-4">Lantronix Scale Reader</h1>
+      <div class="container">
+        <div class="row">
+          <div class="col-md-6">
+            <form class="myForm">
+              <div class="form-group">
+                <label for="userCam">Camera User</label>
```

Comment:
Vsude mas for "userCam"..melo by to odpovidat id kazdeho pole pro ktere je ten label..

Reply:
done